source/client/benchmark_client_impl.h

```diff
@@ -67,12 +69,15 @@ class BenchmarkClientHttpImpl : public BenchmarkClient,
   }
   bool tryStartOne(std::function<void()> caller_completion_callback) override;
   Envoy::Stats::Store& store() const override { return store_; }
-
   // StreamDecoderCompletionCallback
   void onComplete(bool success, const Envoy::Http::HeaderMap& headers) override;
   void onPoolFailure(Envoy::Http::ConnectionPool::PoolFailureReason reason) override;
 
 private:
+  void prefetchPoolConnections() override {
+    dynamic_cast<PrefetchablePool*>(pool_.get())->prefetchConnections();
```

Comment:
Can you just make `pool_` of type `PrefetchablePool` to avoid the dynamic cast?

Reply:
Done in https://github.com/envoyproxy/nighthawk/pull/9/commits/ce508385831a7fb0fb319abf155a2b70011aae2a; does this look better?